docs/examples/use_cases/tensorflow/efficientdet/requirements.txt

```diff
@@ -3,3 +3,4 @@ argparse-utils
 absl-py
 Pillow
 PyYAML
+numpy<1.24
```

Comment:
Why not `<2`? Also, the description of the PR says "pin to 1.24" whereas it actually pins it to _less than_ 1.24.

Reply:
Updated the commit message.